sdks/java/io/hcatalog/src/main/java/org/apache/beam/sdk/io/hcatalog/HCatalogIO.java

```diff
@@ -186,6 +216,26 @@ public Read withFilter(String filter) {
       return toBuilder().setFilter(filter).build();
     }
 
+    /** Sets the duration after which the PartitionPollerFn resumes polling for new partitions. */
```

Comment:
This documentation ideally should be phrased in terms that make sense to a client, rather than reference implementation details. E.g. "If specified, poll for new partitions at this periodicity. Then the returned PCollection will be unbounded. If not specified, (what happens then?)"